app/src/main/java/org/oppia/android/app/viewmodel/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "observable_array_list",
+    srcs = [
+        "ObservableArrayList.kt",
```

Comment:
Here & below: do we need to exclude these source files from the top level app module library (since it's a glob)?

Reply:
Just checked it. This is excluded from the glob
https://github.com/oppia/oppia-android/blob/5c5328ab0dbca246a6c5f90e109fe093c17bd338/app/BUILD.bazel#L414
https://github.com/oppia/oppia-android/blob/5c5328ab0dbca246a6c5f90e109fe093c17bd338/app/BUILD.bazel#L407

Reply:
I don't think that's quite right. None of these files are in any of those lists--how would they be excluded?

Reply:
@BenHenning I've removed these from the Viewmodel.

Reply:
From discussion thread on Hangouts:

The problem I see is that the app library is globing every file in app module minus the lists you outlined. None of the lists include the files that you're moving to a separate Bazel package. That means they will be included in the top-level app library by default.

I could be missing this, but I searched in your PR's app/BUILD.bazel for 'Observable' and did not see the files in any of the lists being excluded from the app library. Am I missing something?